src/applications/vaos/appointment-list/components/UpcomingAppointmentsList.jsx

```diff
@@ -101,7 +101,6 @@ export default function UpcomingAppointmentsList() {
               id={`appointment_list_${monthDate.format('YYYY-MM')}`}
```

Comment:
Hmm I think this should be removed as well. Same with `aria-labelledby` on line 108.

Reply:
I meant the id here as well